eng/tools/BaselineGenerator/baseline.xml

```diff
@@ -3,17 +3,24 @@
   <Package Id="dotnet-sql-cache" Version="2.2.0" />
   <Package Id="dotnet-user-secrets" Version="2.2.0" />
   <Package Id="dotnet-watch" Version="2.2.0" />
+  <Package Id="Microsoft.AspNetCore.Connections.Abstractions" Version="2.2.0" />
   <Package Id="Microsoft.AspNetCore.Cryptography.Internal" Version="2.2.0" />
   <Package Id="Microsoft.AspNetCore.Cryptography.KeyDerivation" Version="2.2.0" />
-  <Package Id="Microsoft.AspNetCore.DataProtection" Version="2.2.0" />
   <Package Id="Microsoft.AspNetCore.DataProtection.Abstractions" Version="2.2.0" />
   <Package Id="Microsoft.AspNetCore.DataProtection.AzureKeyVault" Version="2.2.0" />
   <Package Id="Microsoft.AspNetCore.DataProtection.AzureStorage" Version="2.2.0" />
   <Package Id="Microsoft.AspNetCore.DataProtection.EntityFrameworkCore" Version="2.2.0" />
   <Package Id="Microsoft.AspNetCore.DataProtection.Extensions" Version="2.2.0" />
   <Package Id="Microsoft.AspNetCore.DataProtection.StackExchangeRedis" Version="2.2.0" />
   <Package Id="Microsoft.AspNetCore.DataProtection.SystemWeb" Version="2.2.0" />
+  <Package Id="Microsoft.AspNetCore.DataProtection" Version="2.2.0" />
```

Comment:
What changed in the sorting rules such that the base package is now last?

Reply:
I was manually sorting before and ran the sort lines command in VS Code. As it gets longer, manually sorting will be a pain